plone-6.0.x.cfg

```
@@ -20,3 +20,4 @@ robotframework-assertion-engine = 2.0.0
 robotframework-debuglibrary = 2.3.0
 robotframework-pythonlibcore = 4.2.0
 grpcio-tools = 1.59.0
+docutils = 0.21.2
```

See https://github.com/pypa/readme_renderer/blob/main/CHANGES.rst

```suggestion
readme_renderer <= 43.0
```